scripts/get.py

```diff
@@ -159,6 +159,10 @@ def find_section_definitions(
     """Find definitions from the given *section*, with eventual sub-definitions."""
     definitions: List[Definitions] = []
 
+    # do not look for definions in french verb form section
+    if locale == "fr" and section.title.strip() == "{{S|verbe|fr|flexion}}":
```

Comment:
For multiple flexions, we could do it that way:
```suggestion
    if locale == "fr" and section.title.strip().startswith("{{S|verbe|fr|flexion"):
```